File: docs/samplegeneratedcode/java/sampletargetj/Bean1TestContext.java
package sampletargetj;        
// Factory generated on Tue Jul 02 09:12:42 BST 2024
//   by com.typedpath.unittestof.processor.TestableRoots2Kotlin
//   for com.typedpath.unittestof.annotationprocessor.UnitTestOfAnnotationProcessor    

import com.typedpath.unittestof.testutil.BeanContext;    
public class Bean1TestContext extends BeanContext<Bean1> {
    public final sampletargetj.IService1ProxyContext service1Recorder = new sampletargetj.IService1ProxyContext(getCallLog(), "service1");
    public final sampletargetj.IService1 service1 = service1Recorder.getProxy();  
    
    final Bean1 target= new Bean1(service1Recorder.getProxy());
    @Override
    public Bean1 getTarget() {
        return target;
    }
     
}
    

File: docs/samplegeneratedcode/kotlin/targetsample/Bean1TestContext.kt
package targetsample        
// Factory generated on Tue Jul 02 09:08:31 BST 2024
//   by com.typedpath.unittestof.processor.TestableRoots2Kotlin
//   for UnitTestOfKspSymbolProcessor    

import com.typedpath.unittestof.testutil.BeanContext    
class Bean1TestContext : BeanContext<Bean1>() {
    val service1Recorder = targetsample.IService1ProxyContext(callLog, "service1")
    val service1 = service1Recorder.proxy 
    
    override val target = Bean1(service1Recorder.proxy) 
}
    